src/WebJobs.Script.WebHost/ContainerManagement/AtlasContainerInitializationHostedService.cs

```diff
@@ -9,74 +9,31 @@
 using Azure.Storage.Blobs;
 using Microsoft.Azure.WebJobs.Script.WebHost.Management;
 using Microsoft.Azure.WebJobs.Script.WebHost.Models;
-using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
-using Newtonsoft.Json;
 
 namespace Microsoft.Azure.WebJobs.Script.WebHost.ContainerManagement
 {
-    public class LinuxContainerInitializationHostService : IHostedService
+    public class AtlasContainerInitializationHostedService : LinuxContainerInitializationHostedService
     {
         private readonly IEnvironment _environment;
         private readonly IInstanceManager _instanceManager;
         private readonly ILogger _logger;
         private readonly StartupContextProvider _startupContextProvider;
         private CancellationToken _cancellationToken;
 
-        public LinuxContainerInitializationHostService(IEnvironment environment, IInstanceManager instanceManager, ILogger<LinuxContainerInitializationHostService> logger, StartupContextProvider startupContextProvider)
+        public AtlasContainerInitializationHostedService(IEnvironment environment, IInstanceManager instanceManager,
+            ILogger<LinuxContainerInitializationHostedService> logger, StartupContextProvider startupContextProvider)
```

Comment:
I think the base class should just take an ILogger, which will then allow subclasses to provide a type specific logger

Reply:
Changed 👍

Reply:
Was expecting the logger would be ILogger<AtlasContainerInitializationHostedService> logger now?

Reply:
Sorry, just realized the needed change. Fixed!